setup.py

```diff
@@ -19,7 +19,7 @@
         "pytest",
         "pytest-asyncio",
         "httpx",
-        "pypgstac==0.6.6",
+        "pypgstac==0.7.1",
```

Comment:
you may need to update https://github.com/NASA-IMPACT/veda-backend/blob/develop/docker-compose.yml#L98 to 0.7.1

Reply:
https://github.com/NASA-IMPACT/veda-backend/pull/159/commits/d3c249309ffe0c3fc26dabdac506813f108b849d 🙈

Reply:
I was hoping that the 0.7.1 pypgstac wouldn't have problems with writing to a minor version below but hadn't looked very closely at the container we use for the tests. 

This seems like a good solution for us because we may want to take advantage of some of the other upgrades in 0.7.* pgstac anyway. #154